linker/linker_test.go

```diff
@@ -2041,44 +2038,6 @@ func TestLinkerValidation(t *testing.T) {
 			},
 			expectedErr: `test.proto:1:11: edition value "2022" not recognized; should be one of ["2023"]`,
 		},
-		"failure_use_of_features_without_editions": {
```

Comment:
This moved into the parser's descriptor validation, so is done in parser tests instead of in linker tests.